### PR TITLE
Update update-load_sku_packs.py

### PR DESCRIPTION
**Pull request by gpaulos**
This is a resubmit of PR#582. It had conflicts that I could not easily resolve.

@hohene @jimturnquist @johren @larry-dean

### CODE REVIEW

File: test/util/load_sku_packs.py [hunk 1]
```diff
@@ -35,40 +39,37 @@ def test01_preload_sku_packs(self):
...
+                if skus not in ["debianstatic", ".git", "packagebuild", "tarballs"] and \
+                   os.path.isfile('on-skupack/' + skus + '/config.json'):
+                    try:
+                        configfile = fit_common.json.loads(open("on-skupack/" + skus  + "/config.json").read())
+                        # check if sku pack got installed
+                        if configfile['name'] not in skulist:
+                            print "FAILURE - Missing SKU: " + configfile['name']
+                            error_message += "  Missing SKU: " + configfile['name']
+                    except:
+                        # Check is the sku pack config.json file is valid format, fails skupack install if invalid
```
Comment (jimturnquist) [inline test/util/load_sku_packs.py]: No biggie.   "Check if" not "Check is"

File: test/util/load_sku_packs.py [hunk 2]
```diff
@@ -2,30 +2,34 @@
...
-        # Load SKU packs from GutHub
+from nose.plugins.attrib import attr
+@attr(all=True)
+
+class fit_template(fit_common.unittest.TestCase):
+
+    def test01_download_sku_packs(self):
+        # Download SKU packs from GitHub
         subprocess.call("rm -rf temp.sku; rm -rf on-skupack", shell=True)
         os.mkdir("on-skupack")
```
Comment (johren) [inline test/util/load_sku_packs.py]: Should check the return of this call to make sure it succeeded since much of the rest of the tests depends on this.

Comment (gpaulos) [inline test/util/load_sku_packs.py]: Added that check in last coimmit

Comment (gpaulos) [inline test/util/load_sku_packs.py]: @johren  Added that check in last coimmit

Comment (gpaulos) [inline test/util/load_sku_packs.py]: @johren  Added that check in last commit

**Comment by johren** [comment]
Ready to merge after green on PR builds

**Pull request merged**